FindMe/Tab Bar View Contollers/MainTabBarViewController.m

```diff
@@ -20,4 +26,39 @@ - (void)viewDidLoad
     self.selectedIndex = 1;
 }
 
+- (IBAction)onTapAddContact:(id)sender
+{
+    CNContactPickerViewController *contactPicker = [CNContactPickerViewController new];
+    contactPicker.delegate = self;
+    [self presentViewController:contactPicker animated:YES completion:nil];
+}
+
+- (void)contactPicker:(CNContactPickerViewController *)picker didSelectContacts:(NSArray<CNContact*> *)contacts
+{
+    if (contacts.count >= 1) {
+        [Contact uploadContacts:contacts
+                 withCompletion:^(BOOL succeeded, NSError * _Nullable error) {
+            if (error != nil) {
+                NSLog(@"error %@", error.localizedDescription);
+            }
+            else {
```

Comment:
Seems weird but I would also check the value of succeeded. I'm not sure if you could ever have false with no error but the method signature implies that it's possible.